Started GET "/" for 192.168.0.1 at 2021-02-17 00:45:08 +0000
  
ActiveRecord::ConnectionNotEstablished (FATAL:  role "root" does not exist
):
  
activerecord (6.1.2.1) lib/active_record/connection_adapters/postgresql_adapter.rb:83:in `rescue in new_client'
activerecord (6.1.2.1) lib/active_record/connection_adapters/postgresql_adapter.rb:77:in `new_client'
activerecord (6.1.2.1) lib/active_record/connection_adapters/postgresql_adapter.rb:37:in `postgresql_connection'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:882:in `public_send'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:882:in `new_connection'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:926:in `checkout_new_connection'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:905:in `try_to_checkout_new_connection'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:866:in `acquire_connection'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:588:in `checkout'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:428:in `connection'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:1128:in `retrieve_connection'
activerecord (6.1.2.1) lib/active_record/connection_handling.rb:327:in `retrieve_connection'
activerecord (6.1.2.1) lib/active_record/connection_handling.rb:283:in `connection'
activerecord (6.1.2.1) lib/active_record/migration.rb:611:in `connection'
activerecord (6.1.2.1) lib/active_record/migration.rb:606:in `build_watcher'
activerecord (6.1.2.1) lib/active_record/migration.rb:588:in `block in call'
activerecord (6.1.2.1) lib/active_record/migration.rb:587:in `synchronize'
activerecord (6.1.2.1) lib/active_record/migration.rb:587:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.1.2.1) lib/active_support/callbacks.rb:98:in `run_callbacks'
actionpack (6.1.2.1) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/debug_exceptions.rb:29:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.1.2.1) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.1.2.1) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.1.2.1) lib/active_support/tagged_logging.rb:99:in `block in tagged'
activesupport (6.1.2.1) lib/active_support/tagged_logging.rb:37:in `tagged'
activesupport (6.1.2.1) lib/active_support/tagged_logging.rb:99:in `tagged'
railties (6.1.2.1) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/request_id.rb:26:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.1.2.1) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/static.rb:24:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/host_authorization.rb:98:in `call'
railties (6.1.2.1) lib/rails/engine.rb:539:in `call'
puma (5.2.1) lib/puma/configuration.rb:247:in `call'
puma (5.2.1) lib/puma/request.rb:76:in `block in handle_request'
puma (5.2.1) lib/puma/thread_pool.rb:337:in `with_force_shutdown'
puma (5.2.1) lib/puma/request.rb:75:in `handle_request'
puma (5.2.1) lib/puma/server.rb:431:in `process_client'
puma (5.2.1) lib/puma/thread_pool.rb:145:in `block in spawn_thread'
Started GET "/favicon.ico" for 192.168.0.1 at 2021-02-17 00:45:09 +0000
  
ActiveRecord::ConnectionNotEstablished (FATAL:  role "root" does not exist
):
  
activerecord (6.1.2.1) lib/active_record/connection_adapters/postgresql_adapter.rb:83:in `rescue in new_client'
activerecord (6.1.2.1) lib/active_record/connection_adapters/postgresql_adapter.rb:77:in `new_client'
activerecord (6.1.2.1) lib/active_record/connection_adapters/postgresql_adapter.rb:37:in `postgresql_connection'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:882:in `public_send'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:882:in `new_connection'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:926:in `checkout_new_connection'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:905:in `try_to_checkout_new_connection'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:866:in `acquire_connection'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:588:in `checkout'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:428:in `connection'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:1128:in `retrieve_connection'
activerecord (6.1.2.1) lib/active_record/connection_handling.rb:327:in `retrieve_connection'
activerecord (6.1.2.1) lib/active_record/connection_handling.rb:283:in `connection'
activerecord (6.1.2.1) lib/active_record/migration.rb:611:in `connection'
activerecord (6.1.2.1) lib/active_record/migration.rb:606:in `build_watcher'
activerecord (6.1.2.1) lib/active_record/migration.rb:588:in `block in call'
activerecord (6.1.2.1) lib/active_record/migration.rb:587:in `synchronize'
activerecord (6.1.2.1) lib/active_record/migration.rb:587:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.1.2.1) lib/active_support/callbacks.rb:98:in `run_callbacks'
actionpack (6.1.2.1) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/debug_exceptions.rb:29:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.1.2.1) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.1.2.1) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.1.2.1) lib/active_support/tagged_logging.rb:99:in `block in tagged'
activesupport (6.1.2.1) lib/active_support/tagged_logging.rb:37:in `tagged'
activesupport (6.1.2.1) lib/active_support/tagged_logging.rb:99:in `tagged'
railties (6.1.2.1) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/request_id.rb:26:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.1.2.1) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/static.rb:24:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/host_authorization.rb:98:in `call'
railties (6.1.2.1) lib/rails/engine.rb:539:in `call'
puma (5.2.1) lib/puma/configuration.rb:247:in `call'
puma (5.2.1) lib/puma/request.rb:76:in `block in handle_request'
puma (5.2.1) lib/puma/thread_pool.rb:337:in `with_force_shutdown'
puma (5.2.1) lib/puma/request.rb:75:in `handle_request'
puma (5.2.1) lib/puma/server.rb:431:in `process_client'
puma (5.2.1) lib/puma/thread_pool.rb:145:in `block in spawn_thread'
Started GET "/" for 192.168.0.1 at 2021-02-17 00:46:10 +0000
  
ActiveRecord::NoDatabaseError (FATAL:  database "petstore_backend_development" does not exist
):
  
activerecord (6.1.2.1) lib/active_record/connection_adapters/postgresql_adapter.rb:81:in `rescue in new_client'
activerecord (6.1.2.1) lib/active_record/connection_adapters/postgresql_adapter.rb:77:in `new_client'
activerecord (6.1.2.1) lib/active_record/connection_adapters/postgresql_adapter.rb:37:in `postgresql_connection'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:882:in `public_send'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:882:in `new_connection'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:926:in `checkout_new_connection'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:905:in `try_to_checkout_new_connection'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:866:in `acquire_connection'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:588:in `checkout'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:428:in `connection'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:1128:in `retrieve_connection'
activerecord (6.1.2.1) lib/active_record/connection_handling.rb:327:in `retrieve_connection'
activerecord (6.1.2.1) lib/active_record/connection_handling.rb:283:in `connection'
activerecord (6.1.2.1) lib/active_record/migration.rb:611:in `connection'
activerecord (6.1.2.1) lib/active_record/migration.rb:606:in `build_watcher'
activerecord (6.1.2.1) lib/active_record/migration.rb:588:in `block in call'
activerecord (6.1.2.1) lib/active_record/migration.rb:587:in `synchronize'
activerecord (6.1.2.1) lib/active_record/migration.rb:587:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.1.2.1) lib/active_support/callbacks.rb:98:in `run_callbacks'
actionpack (6.1.2.1) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/debug_exceptions.rb:29:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.1.2.1) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.1.2.1) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.1.2.1) lib/active_support/tagged_logging.rb:99:in `block in tagged'
activesupport (6.1.2.1) lib/active_support/tagged_logging.rb:37:in `tagged'
activesupport (6.1.2.1) lib/active_support/tagged_logging.rb:99:in `tagged'
railties (6.1.2.1) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/request_id.rb:26:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.1.2.1) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/static.rb:24:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/host_authorization.rb:98:in `call'
railties (6.1.2.1) lib/rails/engine.rb:539:in `call'
puma (5.2.1) lib/puma/configuration.rb:247:in `call'
puma (5.2.1) lib/puma/request.rb:76:in `block in handle_request'
puma (5.2.1) lib/puma/thread_pool.rb:337:in `with_force_shutdown'
puma (5.2.1) lib/puma/request.rb:75:in `handle_request'
puma (5.2.1) lib/puma/server.rb:431:in `process_client'
puma (5.2.1) lib/puma/thread_pool.rb:145:in `block in spawn_thread'
Started GET "/favicon.ico" for 192.168.0.1 at 2021-02-17 00:46:11 +0000
  
ActiveRecord::NoDatabaseError (FATAL:  database "petstore_backend_development" does not exist
):
  
activerecord (6.1.2.1) lib/active_record/connection_adapters/postgresql_adapter.rb:81:in `rescue in new_client'
activerecord (6.1.2.1) lib/active_record/connection_adapters/postgresql_adapter.rb:77:in `new_client'
activerecord (6.1.2.1) lib/active_record/connection_adapters/postgresql_adapter.rb:37:in `postgresql_connection'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:882:in `public_send'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:882:in `new_connection'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:926:in `checkout_new_connection'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:905:in `try_to_checkout_new_connection'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:866:in `acquire_connection'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:588:in `checkout'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:428:in `connection'
activerecord (6.1.2.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:1128:in `retrieve_connection'
activerecord (6.1.2.1) lib/active_record/connection_handling.rb:327:in `retrieve_connection'
activerecord (6.1.2.1) lib/active_record/connection_handling.rb:283:in `connection'
activerecord (6.1.2.1) lib/active_record/migration.rb:611:in `connection'
activerecord (6.1.2.1) lib/active_record/migration.rb:606:in `build_watcher'
activerecord (6.1.2.1) lib/active_record/migration.rb:588:in `block in call'
activerecord (6.1.2.1) lib/active_record/migration.rb:587:in `synchronize'
activerecord (6.1.2.1) lib/active_record/migration.rb:587:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.1.2.1) lib/active_support/callbacks.rb:98:in `run_callbacks'
actionpack (6.1.2.1) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/debug_exceptions.rb:29:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.1.2.1) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.1.2.1) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.1.2.1) lib/active_support/tagged_logging.rb:99:in `block in tagged'
activesupport (6.1.2.1) lib/active_support/tagged_logging.rb:37:in `tagged'
activesupport (6.1.2.1) lib/active_support/tagged_logging.rb:99:in `tagged'
railties (6.1.2.1) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/request_id.rb:26:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.1.2.1) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/static.rb:24:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.1.2.1) lib/action_dispatch/middleware/host_authorization.rb:98:in `call'
railties (6.1.2.1) lib/rails/engine.rb:539:in `call'
puma (5.2.1) lib/puma/configuration.rb:247:in `call'
puma (5.2.1) lib/puma/request.rb:76:in `block in handle_request'
puma (5.2.1) lib/puma/thread_pool.rb:337:in `with_force_shutdown'
puma (5.2.1) lib/puma/request.rb:75:in `handle_request'
puma (5.2.1) lib/puma/server.rb:431:in `process_client'
puma (5.2.1) lib/puma/thread_pool.rb:145:in `block in spawn_thread'
  [1m[35m (127.2ms)[0m  [1m[35mCREATE DATABASE "petstore_backend_development" ENCODING = 'unicode'[0m
  [1m[35m (103.3ms)[0m  [1m[35mCREATE DATABASE "petstore_backend_test" ENCODING = 'unicode'[0m
Started GET "/" for 192.168.16.1 at 2021-02-17 00:51:18 +0000
Processing by Rails::WelcomeController#index as HTML
  Rendering /usr/local/bundle/gems/railties-6.1.2.1/lib/rails/templates/rails/welcome/index.html.erb
  Rendered /usr/local/bundle/gems/railties-6.1.2.1/lib/rails/templates/rails/welcome/index.html.erb (Duration: 15.6ms | Allocations: 290)
Completed 200 OK in 61ms (Views: 41.4ms | ActiveRecord: 0.0ms | Allocations: 1557)


Started GET "/favicon.ico" for 192.168.16.1 at 2021-02-17 00:51:19 +0000
  
ActionController::RoutingError (No route matches [GET] "/favicon.ico"):
  
Started GET "/" for 192.168.16.1 at 2021-02-17 00:58:42 +0000
Processing by Rails::WelcomeController#index as HTML
  Rendering /usr/local/bundle/gems/railties-6.1.2.1/lib/rails/templates/rails/welcome/index.html.erb
  Rendered /usr/local/bundle/gems/railties-6.1.2.1/lib/rails/templates/rails/welcome/index.html.erb (Duration: 25.5ms | Allocations: 290)
Completed 200 OK in 65ms (Views: 48.2ms | ActiveRecord: 0.0ms | Allocations: 1557)


Started GET "/" for 192.168.16.1 at 2021-02-17 00:59:04 +0000
Processing by Rails::WelcomeController#index as HTML
  Rendering /usr/local/bundle/gems/railties-6.1.2.1/lib/rails/templates/rails/welcome/index.html.erb
  Rendered /usr/local/bundle/gems/railties-6.1.2.1/lib/rails/templates/rails/welcome/index.html.erb (Duration: 6.5ms | Allocations: 178)
Completed 200 OK in 10ms (Views: 9.3ms | ActiveRecord: 0.0ms | Allocations: 718)


Started GET "/" for 192.168.16.1 at 2021-02-17 01:03:36 +0000
Processing by Rails::WelcomeController#index as HTML
  Rendering /usr/local/bundle/gems/railties-6.1.2.1/lib/rails/templates/rails/welcome/index.html.erb
  Rendered /usr/local/bundle/gems/railties-6.1.2.1/lib/rails/templates/rails/welcome/index.html.erb (Duration: 9.9ms | Allocations: 293)
Completed 200 OK in 42ms (Views: 23.7ms | ActiveRecord: 0.0ms | Allocations: 2269)


